src/components/NavigationBar/index.js

```diff
@@ -165,9 +175,21 @@ class NavigationBar extends Component {
       searchType: this.props.searchType,
       searchSelectWidth: this.getSelectMenuWidth(this.props.searchType),
       showSearchBar: window.innerWidth > 900,
+      chat: false,
     };
   }
 
+  toggleChat = () => {
```

Comment:
These 2 functions should be used on chat icon click based on screen size- 

Check `MessageSection.react.js` file.
```
  openFullScreen = () => {
    const { actions } = this.props;
    actions.handleChatBubble({
      chatBubble: 'minimised',
    });
    actions.handleTestSkillExample({ testSkillExampleKey: -1 });
    actions.openModal({
      modalType: 'chatBubble',
      fullScreenChat: true,
    });
  };

  toggleChat = () => {
    const { actions, chatBubble } = this.props;
    actions.handleChatBubble({
      chatBubble: chatBubble === 'bubble' ? 'full' : 'bubble',
    });
    actions.handleTestSkillExample({ testSkillExampleKey: -1 });
  };
```

Reply:
I think we can remove this icon when the screen size is small. As there is less space left in the navigation bar.

Reply:
@mariobehling What are your views?

Reply:
No, the space management and UI for navbar will be done by me in a new PR. Please make the requested changes.

Reply:
@akshatnitd Done